cluster-stamp-bu0001a0008.json

```diff
@@ -474,19 +474,41 @@
                     {
                         "Name": "apw-routing-rules",
                         "properties": {
-                            "RuleType": "Basic",
+                            "RuleType": "PathBasedRouting",
```

Comment:
go back to basic. It is not the same app + from validation standpoint this can be done using multiple dns A records to different IP address. Therefore, it is going to randomly access the different gateways.

Reply:
we could use the remote address from devtools as a option (Edge).

Reply:
done - go back to basic. 

Pending create validation steps